bin/utils/subkey/src/main.rs

```diff
@@ -384,6 +394,24 @@ where
 
 			print_extrinsic(extrinsic);
 		}
+		("insert", Some(matches)) => {
+			let suri = get_uri("suri", &matches)?;
+			let pair = read_pair::<C>(Some(&suri), password)?;
+			let node_url = matches.value_of("node-url").unwrap_or("http://localhost:9933");
+			let key_type = matches.value_of("key-type").ok_or(Error::Static("Key type id is required"))?;
+
+			// Just checking
+			let _key_type_id = sp_core::crypto::KeyTypeId::try_from(key_type)
+				.map_err(|_| Error::Static("Unknown keytype - provide gran/babe/aura/acco/imon/audi"))?;
```

Comment:
I think that list will easily get out of date, maybe better to somehow get all items from the enum and convert them to `Display/Debug` here?

Reply:
Key-type is not strictly defined, any 4-character string is ok
I reduced variants so that it will not get out of date so easily